### PR TITLE
Add JSDoc to fxpay

### PR DESCRIPTION
**Pull request by muffinresearch**
This is an example just to take a look at what JSDoc might work out like. We'd probably want to push this to gh-pages if we keep this format e.g. html output.

To save the need to build it (though feel free to test it) here's the output: http://fxpay.yakshavings.co.uk/

The change to markdown for the readme was necessary to be able to include it on the homepage.

The question is how much do we want sphinx-docs? If we do I'll try a bit more trying to get the jsdoc->rst stuff working an initial trial of that produced empty .rst files :(

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ node_modules
 
 # For gh-pages
 .ghpages
+docs/*
```
Comment (kumar303) [inline .gitignore]: would this make sense in ``build/docs`` ? That way it would be automatically ignored like other artifacts.

**Comment by kumar303** [comment]
Should we also ignore the docs dir from the bower dist to keep the download small?

**Comment by kumar303** [comment]
this looks great! I like the output a lot. I agree, let's get it on github pages (could be in a follow-up).

r+wc

**Comment by muffinresearch** [comment]
> Should we also ignore the docs dir from the bower dist to keep the download small?

Yep for sure.

File: .gitignore (hunk 1 above)
Comment (muffinresearch) [inline .gitignore]: Yeah that's a good idea. I'll move it around.

**Pull request merged**